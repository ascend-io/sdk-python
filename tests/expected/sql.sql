SELECT A, B, C FROM D