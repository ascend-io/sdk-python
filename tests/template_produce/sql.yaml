id: sql
component:
  transform:
    operator:
      sqlQuery:
        sql: SELECT A, B, C FROM D
