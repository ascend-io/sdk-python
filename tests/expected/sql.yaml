# id: sql
{%- set ascendSqlStmt = ascend_include_raw(resourceParentDirectory, "sql.sql") %}
component:
  transform:
    operator:
      sqlQuery:
        sql: {{ ascendSqlStmt | dumpJson }}
